plugins/jupyter-integration/src/main/java/org/polypheny/db/jupyter/MimetypeFromExtension.java

```diff
@@ -0,0 +1,232 @@
+        extensions.put( "m3u", "audio/x-mpegurl" );
+        extensions.put( "man", "application/x-troff-man" );
+        extensions.put( "me", "application/x-troff-me" );
+        extensions.put( "mesh", "model/mesh" );
+        extensions.put( "mid", "audio/midi" );
+        extensions.put( "midi", "audio/midi" );
+        extensions.put( "mif", "application/x-mif" );
+        extensions.put( "mime", "www/mime" );
+        extensions.put( "mjs", "text/javascript" );
+        extensions.put( "movie", "video/x-sgi-movie" );
+        extensions.put( "mov", "video/quicktime" );
+        extensions.put( "mp2", "audio/mpeg" );
+        extensions.put( "mp2", "video/mpeg" );
+        extensions.put( "mp3", "audio/mpeg" );
+        extensions.put( "mpeg", "video/mpeg" );
+        extensions.put( "mpe", "video/mpeg" );
+        extensions.put( "mpga", "audio/mpeg" );
+        extensions.put( "mpg", "video/mpeg" );
+        extensions.put( "ms", "application/x-troff-ms" );
+        extensions.put( "msh", "model/mesh" );
+        extensions.put( "nc", "application/x-netcdf" );
+        extensions.put( "oda", "application/oda" );
+        extensions.put( "ogg", "application/ogg" );
+        extensions.put( "ogm", "application/ogg" );
+        extensions.put( "pbm", "image/x-portable-bitmap" );
+        extensions.put( "pdb", "chemical/x-pdb" );
+        extensions.put( "pdf", "application/pdf" );
+        extensions.put( "pgm", "image/x-portable-graymap" );
+        extensions.put( "pgn", "application/x-chess-pgn" );
+        extensions.put( "pgp", "application/pgp" );
+        extensions.put( "pl", "application/x-perl" );
+        extensions.put( "pm", "application/x-perl" );
+        extensions.put( "png", "image/png" );
+        extensions.put( "pnm", "image/x-portable-anymap" );
+        extensions.put( "pot", "application/mspowerpoint" );
+        extensions.put( "ppm", "image/x-portable-pixmap" );
+        extensions.put( "pps", "application/mspowerpoint" );
+        extensions.put( "ppt", "application/mspowerpoint" );
+        extensions.put( "ppz", "application/mspowerpoint" );
+        extensions.put( "pre", "application/x-freelance" );
+        extensions.put( "prt", "application/pro_eng" );
+        extensions.put( "ps", "application/postscript" );
+        extensions.put( "qt", "video/quicktime" );
+        extensions.put( "ra", "audio/x-realaudio" );
+        extensions.put( "ram", "audio/x-pn-realaudio" );
+        extensions.put( "rar", "application/x-rar-compressed" );
+        extensions.put( "ras", "image/cmu-raster" );
+        extensions.put( "ras", "image/x-cmu-raster" );
+        extensions.put( "rgb", "image/x-rgb" );
+        extensions.put( "rm", "audio/x-pn-realaudio" );
+        extensions.put( "roff", "application/x-troff" );
+        extensions.put( "rpm", "audio/x-pn-realaudio-plugin" );
+        extensions.put( "rtf", "application/rtf" );
+        extensions.put( "rtf", "text/rtf" );
```

Comment:
@gartens Is this duplicate intentional.

Reply:
I kept them, so it would be easier to sync with upstream.  Looks like they (fixed)[https://sqlite.org/althttpd/info/c0bdc68e6c56ef25] the duplicates just after I copied the table.